eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -27,11 +27,18 @@ steps:
         displayName: Build host packages
 
   # Build coreclr native test output
+  # For Wasm, we are passing the osGroup where normally the archType would be so we can set the right __DistroRid in src/tests/build.cmd
```

Comment:
To comment does not seem right - we are not passing `osGroup` AFAICT.

Reply:
Removed